llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1447,6 +1447,74 @@ LogicalResult TopkOp::getResultTilePosition(
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// PackOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult PackOp::verify() {
+  Operation *op = getOperation();
+  if (getNumInputs() != 1)
+    return op->emitOpError("expects only 1 input operand");
+  if (getNumOutputs() != 1)
+    return op->emitOpError("expects only 1 output operand");
+  AffineMap packMap = getPackMap();
+  if (packMap.getNumDims() != getOutputRank())
```

Comment:
Probably need to build on this a bit. I think the input is of rank r,  the output rank is 2*r. 
Also the number of tile values specified should be `r`.